docs/Using-Fleet/FAQ.md

```diff
@@ -318,3 +336,27 @@ Run `sudo apt remove fleet-osquery -y`
 
 ### CentOS
 Run `sudo rpm -e fleet-osquery-X.Y.Z.x86_64`
+
+## How does Fleet determines online and offline status?
+
+### Online hosts
+
+**Online** hosts are devices that are connected to Fleet and will respond to a live query.
+
+Fleet considers a device as online if it has connected successfully in a window of time set by `distributed_interval` (or `config_tls_refresh`, whichever is smaller).
+A buffer of 60 seconds is added to the calculation to avoid unnecessary flapping between online/offline status (in case hosts take a bit longer than expected to connect to Fleet).
+The values for `distributed_interval` and `config_tls_refresh` can be found in the **Settings > Organization settings > Agent options** page for global hosts
+and in the **Settings > Teams > TEAM NAME > Agent options** page for hosts that belong to a team.
+
+For example:
+
+`distributed_interval=10, config_tls_refresh=30`
+A host is considered online if it has connected to Fleet in the last 70 (10+60) seconds.
+
+`distributed_interval=30, config_tls_refresh=20`
+A host is considered online if it has connected to Fleet in the last 80 (20+60) seconds.
+
+### Offline hosts
+
+**Offline** hosts are devices that are not online. These hosts may be devices that are turned off or not connected to the internet.
```

Comment:
```suggestion
**Offline** hosts won't respond to a live query. These hosts may be shut down, asleep, or not connected to the internet.
```

Suggested edits based on the following changes coming to the Fleet UI:
![Screen Shot 2022-09-15 at 9 21 05 AM](https://user-images.githubusercontent.com/47070608/190414545-ce23e5c6-0bdc-4910-af24-ac594ab7b21f.png)